rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly have Getters and Setters for @book and @person. You could work it out like this - `attr_accessor :date, :book, :person` 👍🏽